typescript/README.md

```diff
@@ -0,0 +1,9 @@
+name: TypeScript
+
+description: A typed superset of JavaScript.
+
+color: 007acd
+
+icon: https://img.enkipro.com/0090595b46a8da63f30e282404e449e4.png
+
+language: TypeScript
```

Comment:
This is not even used anymore i think, given all code snippets have a language tag; don't remove for now but it's a note to ourselves to remove it in the future @Loopiezlol

Reply:
yes, it's not used anymore, but maybe we can use it as a fallback for all code snippets within the topic

right now we default to `bash`: https://github.com/enkidevs/curriculum-tools/blob/master/lib/insight.js#L29

Reply:
cool, im migrating that repo to curriculum-parsers and will change it to `plain-text` for now, which is supported in the app